WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/ServiceRequestContext.cs

```diff
@@ -179,11 +249,16 @@ public bool IsNewSessionId(string sessionId)
                                                    fileName),
                                       requestTaskOptions.ApplicationNamespace);
 
-    if (ServicesMapper.ContainsKey(serviceId.Key))
+    if (currentService_?.ServiceId == serviceId)
     {
-      return serviceId;
+      return currentService_;
     }
 
+    logger_.LogInformation($"Worker needs to load new context, from {currentService_?.ServiceId?.ToString() ?? "null"} to {serviceId}");
```

Comment:
`LogDebug`?

Reply:
May be an important metric, so I think we should keep it as Info